Scripts/Build-Phases/sparkle-code-sign.sh

```diff
@@ -55,5 +55,5 @@ echo "Signing... "
 ${CODESIGN_TOOL} "${IDENTITY}" "${XPC_TARGET}/${XPC_NETWORK}"
 ${CODESIGN_TOOL} "${IDENTITY}" "${XPC_TARGET}/${XPC_LAUNCHER}"
 ${CODESIGN_TOOL} "${IDENTITY}" "${XPC_TARGET}/${XPC_STATUS}"
-${CODESIGN_HARDENEDRUNTIME_CMD} "${IDENTITY}" "${UPDATER_TARGET}/${SPARKLE_AUTOUPDATE}"
-${CODESIGN_HARDENEDRUNTIME_CMD} "${IDENTITY}" "${UPDATER_TARGET}/${SPARKLE_UPDATER}"
+${CODESIGN_HARDENEDRUNTIME_CMD} -s "${IDENTITY}" "${UPDATER_TARGET}/${SPARKLE_AUTOUPDATE}"
+${CODESIGN_HARDENEDRUNTIME_CMD} -s "${IDENTITY}" "${UPDATER_TARGET}/${SPARKLE_UPDATER}"
```

Comment:
This change is because I removed the `-s` from the `CODESIGN_HARDENEDRUNTIME_CMD` definition above.

I find it's clear like this what the purpose of that `IDENTITY` value is.

I did try to include the whole `-s "${IDENTITY}"` in the definition, but the spaces exploded on me when the script run. I decided not to spend extra time on that.